pkgs/by-name/zo/zotify/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  fetchFromGitHub,
+  python3Packages,
+  protobuf_21,
+  lib,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "zotify";
+  version = "0.16.13";
+
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "zotify-dev";
+    repo = "zotify";
+    rev = "5da27d32a1f522e80a3129c61f939b1934a0824a";
+    hash = "sha256-KA+Q4sk+riaFTybRQ3aO5lgPg4ECZE6G+By+x2uP/VM=";
+  };
+
+  postPatch = ''
+    substituteInPlace ./setup.cfg --replace-fail "protobuf==3.20.1" "protobuf"
```

Comment:
Can't we relax dependencies here instead of doing a substitute?

Reply:
You need to add `python3Packages.pythonRelaxDepsHook` to `nativeBuildInputs` and set `pythonRelaxDeps = [ "protobuf" ];`

Reply:
It's working now, I didn't know you had to add it to `nativeBuildInputs`, thanks! I'll modify it now.

Reply:
Yeah it still fails.

Reply:
If you use https://bin.hubrecht.ovh/?48418a0288c4d03d#5sNySCHkptNWGCnRpfxupgjKq3JdE6Xb1G5w8ovWrQxc it seems to build fine

Reply:
Oh, yeah that builds fine for me too. Thanks, I pushed the new modifications.